api_core/google/api_core/protobuf_helpers.py

```diff
@@ -15,17 +15,19 @@
 """Helpers for :mod:`protobuf`."""
 
 import collections
-try:
-    from collections import abc as collections_abc
-except ImportError:  # Python 2.7
-    import collections as collections_abc
 import copy
 import inspect
 
 from google.protobuf import field_mask_pb2
 from google.protobuf import message
 from google.protobuf import wrappers_pb2
 
+try:
```

Comment:
This seems out of order? Did a linter do this?

Reply:
It was some sort of the `flake8` error and my [build](https://source.cloud.google.com/results/invocations/57246fd0-a187-4c77-be97-85a89b8564b4/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fapi_core) failed a couple of times.
![image](https://user-images.githubusercontent.com/46078689/68569340-f31e8a00-046e-11ea-8d29-92cd7a034a78.png)